cfn/cfn.go

```diff
@@ -181,6 +187,12 @@ func makeTestEventFunc(h Handler) testEventFunc {
 		}
 		request := handler.NewRequest(
 			event.Request.LogicalResourceIdentifier,
+			event.Request.NextToken,
+			"",  //TODO: understand how to fix this to use a proper value
```

Comment:
I think it should be the same as above `event.StackID` and `event.RequestData.StackTags`.